drud/ddev-ui
# setup feedback links and setup alpha notice #112

## The Problem/Issue/Bug:
The app needed to notify users of it's "Alpha" Status. As well as provide links to create issues and provide feedback.

## How this PR Solves The Problem:
It adds a notice to the app thanking them for trying the alpha. It also let's them know how to submit feedback. It also introduces some links in the sidebar footer to submit issues, and links to docs.
<img width="1616" alt="screenshot 2018-09-05 09 17 47" src="https://user-images.githubusercontent.com/192756/45105102-c8110b80-b0f0-11e8-8fec-8871e4cfff66.png">


## Manual Testing Instructions:

Download - https://391-99947022-gh.circle-artifacts.com/0/artifacts/DDEV%20UI-0.4.0-alpha.1.dmg

Open app and make sure you see the welcome message. Follow the links to submit an issue.

## Related Issue Link(s):
#112 

## Release/Deployment notes:
